api/src/pcapi/core/offerers/synchronize_venues_banners_with_google_places.py

```diff
@@ -174,3 +177,35 @@ def synchronize_venues_banners_with_google_places(begin: int, end: int, limit: i
             "nb_places_with_owner_photo": nb_places_with_owner_photo,
         },
     )
+
+
+def delete_venues_banners(begin: int, end: int | None, limit: int | None) -> None:
+    logger.info(
+        "deleting old google places banners",
+        extra={
+            "begin": begin,
+            "end": end,
+            "limit": limit,
+        },
+    )
+    google_places_info_query = offerers_models.GooglePlacesInfo.query.filter(
+        offerers_models.GooglePlacesInfo.venueId >= begin
+    ).order_by(offerers_models.GooglePlacesInfo.venueId)
+    nb_deleted_banners = 0
+    if end is not None:
+        google_places_info_query = google_places_info_query.filter(offerers_models.GooglePlacesInfo.venueId <= end)
+    if limit is not None:
+        google_places_info_query = google_places_info_query.limit(limit)
+    for place_info in google_places_info_query:
+        if place_info.bannerUrl:
+            delete_public_object(GOOGLE_PLACES_BANNER_STORAGE_FOLDER, place_info.bannerUrl.split("/")[-1])
+            nb_deleted_banners += 1
+        place_info.bannerUrl = None
+        place_info.bannerMeta = None
```

Comment:
Ces 2 lignes pourraient être dans le bloc `place_info.bannerUrl`, non ?

Et en fait, est-ce que la condition `bannerUrl is not null` ne pourrait pas être inclue dans la requête SQL ?